lib/gitlab_shell.rb

```diff
@@ -10,6 +10,7 @@ class DisallowedCommandError < StandardError; end
   def initialize
     @key_id = /key-[0-9]+/.match(ARGV.join).to_s
     @origin_cmd = ENV['SSH_ORIGINAL_COMMAND']
+    @client_ip = ENV['SSH_CONNECTION'].split(" ")[0]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.